src/modules/home/components/event-carousel/EventCarousel.vue

```diff
@@ -2,12 +2,21 @@
   <div class="relative rounded-2xl overflow-hidden h-40 w-120 z-20 shadow">
     <event-carousel-slide
       v-for="(event, slideIndex) in testEventProps"
-      :key="event.eventTitle"
+      :key="event.title"
       :slideIndex="slideIndex"
       :currentIndex="currentIndex"
       :transitionAnimation="transitionAnimation"
     >
-      <CardBanner class="w-full" :eventTitle="event.eventTitle" />
+      <CardBanner
+        class="w-full"
+        :eventTitle="event.title"
+        :tags="event.tags"
+        :eventDescription="event.description"
+        :eventDate="event.date"
+        :eventLocation="event.location"
+        :eventTime="event.time"
+        :imgUrl="event.img"
+      />

```

Comment:
Will it be easier to pass the whole event object as prop and let the card banner read the attribute of the event object itself instead of passing each individual attribute as prop? Not a suggestion to change but should be a convention we should discuss together,